distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -84,6 +84,35 @@
                 <exclude>
                     **/commons-collections/commons-collections/3.2.1/commons-collections-3.2.1.jar
                 </exclude>
+                <!--
+                Excluding all Fabric8 Kubernetes jars since they are not needed or enabled and
+                introduce vulnerability CVE-2016-1906.
+                -->
+                <exclude>
+                    **/io/fabric8/kubernetes*/**/*
+                </exclude>
+                <!--
+                Excluding pax-web-jsp 6.0.6 which adds support for JSP but is not currently used
+                and introduces multiple severe vulnerabilities (CVE-2009-3548, CVE-2011-3190,
+                CVE-2016-6325, CVE-2014-0230, CVE-2016-5425, CVE-2016-8735, CVE-2013-2185)
+                -->
+                <exclude>
+                    **/org/ops4j/pax/web/pax-web-jsp/6.0.6/pax-web-jsp-6.0.6.jar
+                </exclude>
+                <!--
+                Excluding all Netty 4.0.33-Final jars which are not used and introduce vulnerability
+                CVE-2016-4970.
+                -->
+                <exclude>
+                    **/io/netty/**/4.0.33.Final/*
```

Comment:
This has already been excluded earlier in this file.

Reply:
Great catch! Removed.